### PR TITLE
[WIP] FAQ page

### PR DESCRIPTION
**Pull request by Remi-Gau**
This is the PR where we can work on the FAQ page.

The way I have set this up is by creating a `_data/faq.yml` where we can add new questions with response and organize them in subsections.

This is then taken up by the`_include/faq.html` to render the page.

Right now the layout is VERY basic and can be greatly improved but hopefully this should be OK.

I suspect that for the content we can still work on this [google doc](https://docs.google.com/document/d/1SLvunhFAF5jfEA0Vq4VlqQQ4MaTKCkNFJwZyHD2eKIc/edit?usp=sharing)

### CODE REVIEW

**Comment by Remi-Gau** [comment]
Tagging you here so to avoid adding noise on the mattermost channel.
@llevitis @sofievdbos @complexbrains

**Comment by llevitis** [comment]
> Tagging you here so to avoid adding noise on the mattermost channel.
> @llevitis @sofievdbos @complexbrains

@Remi-Gau I'm so sorry, I somehow missed this when you tagged me!! Will review now :)

**Comment by llevitis** [comment]
This looks great - I'll copy over some more text and add the page to the top nav bar

**Comment by complexbrains** [comment]
@Remi-Gau same here too I am so sorry, I somehow just have seen it. I will have a look in a couple of days. So sorry for the delay :(

**Comment by complexbrains** [comment]
@llevitis Do I supposed to see the document you listed in your previous link? If so I guess I will require permission if you do not mind please but if it is not publically open yet it is definitely fine. Thank you

**Comment by llevitis** [comment]
> @llevitis Do I supposed to see the document you listed in your previous link? If so I guess I will require permission if you do not mind please but if it is not publically open yet it is definitely fine. Thank you

@complexbrains Just gave you permissions to edit :)

File: _data/faq.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    # - {question: "May I be involved with the OHBM hackathon team?", answer: "OHBM"}
+    # - {question: "Is there any funding available to hackathon attendees?", answer: "OHBM"}
+-
+  group: "Attendees"
+  questions:
+    # - {question: "May I pitch a project? Do I have to? Which procedure do I have to follow?", answer: " "}
+    # - {question: "Am I eligible to attend a Distributed Hackathon event?", answer: " "}
+    - {question: "How do I join a project at a hackathon? ", answer: "In the weeks leading up to the hackathon, project leaders will submit a description of their project. The projects will be listed on a github repository and on the website. So you can start having a look at what projects are lined up and think about which one you want to join. Then at the hackathon each project leader has to pitch his project at the beginning of the event. Then there will be some time for you to chat with the project leaders before you 'decide' which projects to join. But it is a very flexible thing: if you are interested by different projects and wants to do a bit of both: it is OK."}
```
Comment (llevitis) [inline _data/faq.yml:22]: should we perhaps encourage newcomers to only join one project?

Comment (Remi-Gau) [reply _data/faq.yml]: yes good point.
will add this as a comment in the google doc.

I think that the FAQ page is "good enough" in terms of look.

Now I will focus on the content.

**Comment by Remi-Gau** [comment]
FYI: started making sure we use the word "brainhack" and not "hackathon" in the FAQ.
To be consistent

**Comment by Remi-Gau** [comment]
@llevitis And this one is good for me too.
Some questions needs to still be added but I think we are good to start.

File: FAQ.html~beginning of a FAQ page [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (llevitis) [inline FAQ.html~beginning of a FAQ page:1]: was this file meant to be added?

File: .gitignore [hunk 3]
```diff
@@ -14,3 +14,5 @@ ehthumbs.db
 
 ### MAC ###
 *DS_Store
+
+FAQ.html~master
```
Comment (llevitis) [inline .gitignore:18]: is this also supposed to be added?

File: FAQ.html~beginning of a FAQ page (hunk 2 above)
Comment (Remi-Gau) [reply FAQ.html~beginning of a FAQ page:1]: nope. Will clean this up.

File: .gitignore (hunk 3 above)
Comment (Remi-Gau) [reply .gitignore:18]: nope. Will clean this up.

**Comment by Remi-Gau** [comment]
woohoo :-)

**Pull request merged**